### ISSUE TITLE
Create a doctors listing page (conventional routing)

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#37]
### Who

- [x] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

A doctors listing page with summary information from each of the doctors.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #9
